nonio-ios/Sources/APIs/NonioAPI.swift

```diff
@@ -5,9 +5,12 @@ import Combine
 enum NonioAPI {
     case getPosts(GetPostParams)
     case getTags
+    case getComments(id: String)
+    case login(user: String, password: String)
+    case userInfo(user: String)
 }
 
-extension NonioAPI: TargetType {
+extension NonioAPI: TargetType, AccessTokenAuthorizable {
     var baseURL: URL {
         return URL(string: "https://api.non.io")!
```

Comment:
Can we abstract this into a configuration file for the API urls, so that it's easier for people to spin up a clone?

Reply:
Good point, move it to `Configuration` file. Fixed: ca6381864918980470b8a6459d3e27b50d66042b